public/assets/css/admin_formulary.css

```diff
@@ -0,0 +1,51 @@
+:root {
+    --alerte-radius: 5px;
+    --alerte-padding: 10px;
+    --success-bg-color: #c2faa8;
+    --success-font-color: var(--primary-color);
+    --errors-bg-color: rgb(240, 153, 153);
+    --errors-font-color: red;
```

Comment:
3 different ways of using colors => avoid

Reply:
yes correction done